### ISSUE TITLE
Unable to run skhd directly after Sequoia upgrade

### ISSUE THREAD

**Issue by nickcorin** [koekeishiya/skhd#371]
After upgrading to macOS Sequoia, I was prompted to re-grant `skhd` access to accessibility features. It was already enabled, and flipping the toggle off and on did nothing. To rid the prompt appearing when running `skhd --start-service` or `skhd --restart-service`, I had to:

1. Remove the `skhd` entry from the list of applications which have been granted accessibility access.
2. Run `skhd --restart-service`.
3. Grant access to the new entry through following the prompt once more.

However, I still cannot just run `skhd` directly in my terminal. I am constantly met with a prompt to grant accessibility access to either`skhd`, or `alacritty`.  Both of these have been granted permissions, and all has worked fine until the upgrade.

All of this is to debug and test some broken features with Sequoia, specifically to interact with `yabai`, and more specifically, [this](https://developer.apple.com/forums/thread/763878?answerId=804374022#804374022), which seems to have broken every single keybind I have configured in `skhd`.

For clarity, even with `skhd` running as a service, none of my keybinds are working. Assumed due to the link above.

**Comment by maturk** [comment]
Funny temporary fix, I found that typing `skhd -V` in the terminal, and leaving the output open in the terminal returns the hot keys back.

But I can replicate the above issue, currently `skhd --restart-service` is completely stuck

**Comment by enweg** [comment]
I had a similar problem. Doing an install from source into another location than the standard homebrew location and starting the service from Terminal worked for me. I also had to allow Accessibility services for the Terminal app though. Some key bindings for yabai (those moving windows to other spaces) still don't work but this may well be due to yabai and not skhd.

**Comment by RedemptionC** [comment]
reinstalled using brew and it worked for me.

**Comment by Oltho** [comment]
Despite having `skhd`, `iTerm2` and `Terminal` enabled under `Privacy & Security > Accessibility`, I noticed that
- `skhd -V` can be launch from both `ITerm2` and `Terminal`
- when running via `iTerm2`, skhd is working as expected
- when running via `Terminal`, skhd is running but **not working**

I would imagine that launchd is using `Terminal` in the background which might be linked to the issue ?

Can somebody else replicate ?
- macOs: Sequoia 15.2
- skhd: 0.3.9

**Comment by cballenar** [comment]
skhd stopped working after updating to latest (0.3.9) on macOS Sequoia (15.3).

- It's already present in Privacy & Security > Accessibility. 
- I've tried reinstalling a few times, as well as the service
- I haven't been able to run debug mode, it either doesn't seem to do anything or I get `skhd: could not lock pid-file! abort..` (which I get a lot after skhd has been started)

It was working mostly fine before updating to 0.3.9 so not sure when it broke. I don't know what version I was running before but I hadn't updated in months. :/

**Comment by robertherdzik** [comment]
> Funny temporary fix, I found that typing `skhd -V` in the terminal, and leaving the output open in the terminal returns the hot keys back.
> 
> But I can replicate the above issue, currently `skhd --restart-service` is completely stuck

Hi, I have similar issue and indeed leaving terminal open after execution of `skhd -V` work, but is there any other solution to this. Also worth note that if i kill terminal window with `skhd -V` execution only one mapping is working and the rest not - is really interesting. 

Thanks in advance!

**Comment by GregHilston** [comment]
> Funny temporary fix, I found that typing `skhd -V` in the terminal, and leaving the output open in the terminal returns the hot keys back.
> 
> But I can replicate the above issue, currently `skhd --restart-service` is completely stuck

@maturk thanks for this, this work around works for me to. Have you found a permanent fix?

**Comment by daikiejp** [comment]
> Potential solution found under [this comment](https://github.com/koekeishiya/skhd/issues/382#issuecomment-2585714135) in [#382](https://github.com/koekeishiya/skhd/issues/382). I removed skhd from "System Preferences > Privacy & Security > Accessibility". Then I ran `skhd --stop-service` follow up by `rm /tmp/skhd_$USER.pid`. Now start skhd by using `skhd --start-service`, there should be a popup asking for permission, allow it on system preferences. It works for me again even after restart, hope this helps.

Thanks, It's works for me

**Comment by GregHilston** [comment]
skhd version `skhd-v0.3.9`
macos sequoia version `15.4.1`

I cannot repeat success described [here](https://github.com/koekeishiya/skhd/issues/371#issuecomment-2851578812)

Restarting the skhd service does not result in any output, nor having the program start. Running: `$ skhd -V` results in:

```
skhd: could not lock pid-file! abort..
```

**Comment by francois-gu** [comment]
> Potential solution found under [this comment](https://github.com/koekeishiya/skhd/issues/382#issuecomment-2585714135) in [#382](https://github.com/koekeishiya/skhd/issues/382). I removed skhd from "System Preferences > Privacy & Security > Accessibility". Then I ran `skhd --stop-service` follow up by `rm /tmp/skhd_$USER.pid`. Now start skhd by using `skhd --start-service`, there should be a popup asking for permission, allow it on system preferences. It works for me again even after restart, hope this helps.

After a clean install where things did not work, above worked for me too, thanks 🙏

**Comment by francois-gu** [comment]
> > Potential solution found under [this comment](https://github.com/koekeishiya/skhd/issues/382#issuecomment-2585714135) in [#382](https://github.com/koekeishiya/skhd/issues/382). I removed skhd from "System Preferences > Privacy & Security > Accessibility". Then I ran `skhd --stop-service` follow up by `rm /tmp/skhd_$USER.pid`. Now start skhd by using `skhd --start-service`, there should be a popup asking for permission, allow it on system preferences. It works for me again even after restart, hope this helps.
> 
> Thanks, It's works for me

Worked for me too after a clean install (don't know if it would have worked without)

**Comment by souvikmishra** [comment]
Tahoe 26.2 did something similar, because of which I was not able to add accessibility permissions to `skhd` or `yabai` in settings. It just wouldn't allow it. Upgrading to Tahoe 26.3 and following @JinRecords method worked.

**Comment by anuragkumar2921** [comment]
I'm running Tahoe 26.4. I faced the same issue yesterday. After upgrading from yabai-v7.1.15 -> yabai-v7.1.18 and then followed by yabai --start-service , skhd --start-service , ./scripting-addition.sh my issue got resolved. Thanks yabai Team for awesome work.
